packages/web/src/helpers/_links.scss

```
@@ -0,0 +1,21 @@
+@use 'sass:map';
+@use '../settings/links' as links-settings;
+@use '../tools/links';
+
+@include links.variants('.link', links-settings.$variants);
+
+.link-underlined {
+    text-decoration: underline;
+}
+
+// Links in headings cannot be underlined
+[class*='typography-heading'] a {
+    text-decoration: none;
+}
+
+// Only links in headings have visited state
+[class*='typography-heading'] :visited {
+    color: links-settings.$default-visited;
```

While effectively the same, this could get the value directly from the settings map:

```suggestion
    color: map.get(links-settings.$variants, primary, visited);
```

But I`m not sure at all, it's up to you.

---

Yea, we can do this. I noticed that I import and don't use sass:map anyway.